templates/landscape/features.html

```diff
@@ -1,191 +1,243 @@
+            }}
+          </div>
+        </div>
+        <h3 class="p-equal-height-row__item p-heading--5">
+          Regulatory compliance is significantly simplified with custom reporting
+        </h3>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
+
+  <section class="p-section--shallow">
+    {%- call(slot) vf_tiered_list(is_list_full_width_on_tablet=false) -%}
+      {%- if slot == 'title' -%}
+        <h2>Why choose landscape?</h2>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_1' -%}
+        <h3 class="p-heading--5">Control your inventory</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_1' -%}
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Quickly track full software package information for all registered machines</li>
+          <li class="p-list__item is-ticked">Gather asset information in real time</li>
+          <li class="p-list__item is-ticked">Create dynamic search groups to perform operations on categories of machines</li>
+          <li class="p-list__item is-ticked">Easily access any machine property</li>
+        </ul>
+      {%- endif -%}
 
-<section class="p-strip--light">
-  <div class="u-fixed-width">
-    <h2 class="u-sv3">Deploy, monitor and manage your Ubuntu systems</h2>
-    <p>
-      <a href="/landscape/pricing" class="p-button--positive">Get Landscape</a>
-      <a href="#get-in-touch" class="p-button">Contact us</a>
-    </p>
-  </div>
-</section>
+      {%- if slot == 'list_item_title_2' -%}
+        <h3 class="p-heading--5">Package repository management</h3>
+      {%- endif -%}
 
-<!-- Set default Marketo information for contact form below-->
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/landscape" data-form-id="4473" data-lp-id="" data-return-url="/landscape/features" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
-</div>
+      {%- if slot == 'list_item_description_2' -%}
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Mirror and stage internal or external APT repositories</li>
+          <li class="p-list__item is-ticked">Upload and manage custom packages</li>
+        </ul>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_3' -%}
+        <h3 class="p-heading--5">
+          Integrate Landscape
+          <br />
+          with your current systems
+        </h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_3' -%}
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Scriptable API to customize Landscape for your organization’s needs</li>
+          <li class="p-list__item is-ticked">
+            Integrate with your current configuration management, monitoring and ticketing systems
+          </li>
+        </ul>
+      {%- endif -%}
+    {%- endcall -%}
+  </section>
+
+  <hr class="p-rule is-fixed-width" />
+
+  <section class="p-strip is-deep">
```

Comment:
There should be some space between `h2` and CTA block
<img width="867" alt="Screenshot 2024-12-19 at 3 10 34 PM" src="https://github.com/user-attachments/assets/6b666f79-e5b5-4b15-a71c-9ef972b48eb4" />